index.js

```diff
@@ -211,7 +211,10 @@ class Conf {
 
 			if (this.encryptionKey) {
 				try {
-					const decipher = crypto.createDecipher(encryptionAlgorithm, this.encryptionKey);
+					const iv = Buffer.alloc(16);
+					const key = crypto.pbkdf2Sync(this.encryptionKey, iv.toString(), 10000, 32, 'sha512');
```

Comment:
I think this will break existing uses where the `encryptionKey` is not using `pbkdf2Sync`.

Reply:
According to my tries and by https://github.com/nodejs/node/issues/16746 it may impossible to make this upgrade without any harm since `key` patrameter in `createCipherkv` method require fixed length, so any previous key that does not match `createCipherIv` criteria will produce an error :confused: